src/utils/selectFoldersOnly.ts

```diff
@@ -0,0 +1,29 @@
+import getFileTypeFromPath from "./getFileTypeFromPath";
+import { Core } from "../tree/types";
+
+/**  Will be the MarkDownTree with only Folders
+ * @param {Core[]} treeCore  The whole MarkDownTree
+ * @returns {Core[]} the MarkDownTree with only Folders
+ */
+const typeToShow="FOLDER";
+
+const generateTreeCore = (treeCore: Core[]): Core[] => {
+  const foldersOnlyAsLines: Core[] = [];
+
+  treeCore.forEach((core) => {
+    const coreType = getFileTypeFromPath(core.path);
+    if (coreType === typeToShow) {
```

Comment:
@ralph-dev well it used to be (coreType === "FOLDER") but I figured instead of only showing folders we can have a potential for it to show any kind of file we want...
also you said use TypeScript not this string FOLDER
which I didn't get quietly